### PR TITLE
Update guide_nextcloud.rst (Onlyoffice)

### PR DESCRIPTION
**Pull request by ciconi**
Added (a now corrected) instruction for installing Onlyoffice in Nextcloud

### CODE REVIEW

File: source/guide_nextcloud.rst [hunk 1]
```diff
@@ -79,6 +79,19 @@ Additionally, you can choose where Nextcloud is going to store your data files.
...
+To edit text and spreadsheet documents, install these apps from the admin interface: 
+
+* Onlyoffice (the bridge to the Onlyoffice server)
+* Community Document Server (a light version of the Onlyoffice server)
+
+The installation of the Community Document Server probably doesn't work from the admin panel. Then install it from the shell:
+
+::
+
+[isabell@stardust ~]$ cd apps
```
Comment (SalocinHB) [inline source/guide_nextcloud.rst]: this implies ~/apps, which shouldn't exist if Nextcloud was installed to the DocumentRoot according to this guide

File: source/guide_nextcloud.rst [hunk 2]
```diff
@@ -79,6 +79,19 @@ Additionally, you can choose where Nextcloud is going to store your data files.
...
+[isabell@stardust apps]$ curl -L https://github.com/nextcloud/documentserver_community/releases/latest/download/documentserver_community.tar.gz | tar -xvzf -
```
Comment (SalocinHB) [inline source/guide_nextcloud.rst]: Hm, this doesn't install the server. The Onlyoffice settings in Nextcloud now ask for a hostname and port where the server can be reached.

Comment (ciconi) [reply source/guide_nextcloud.rst]: When the document server is installed and enabled, there should be no available options for configuration. I changed some lines, sorry if I made a mess, its my first commit ever :-)

**Comment by luto** [comment]
Works great! Thank you.

**Pull request merged**